### PR TITLE
RTC Rollover Fix

### PR DESCRIPTION
**Pull request by KenwoodFox**

### CODE REVIEW

File: firmware/include/timezones.h [hunk 1]
```diff
@@ -1,48 +1,84 @@
...
+ * @brief Converts raw UTC 24-hour time to 12-hour or 24-hour format.
+ *
+ * @param hour (0 - 23) Input hour in 24-hour format.
+ * @param use_24hr_format Set to true for 24-hour format, false for 12-hour format.
+ * @return int Converted hour in either 12-hour (1 - 12) or 24-hour (1 - 23) format.
+ */
+int meridianTime(int hour, bool use_24hr_format = false)
+{
+    // Handle edge cases
+    if (hour < 0 || hour > 23)
```
Comment (ac1ja) [inline firmware/include/timezones.h]: remove this

File: firmware/lib/display/display.cpp [hunk 2]
```diff
@@ -70,10 +71,10 @@ void Display::setDispTime(uint8_t hour, uint8_t minute, uint8_t second, uint8_t
         dispData = getDigitByte(minute % 10, 3);
         break;
     case 4:
-        dispData = second < 60 ? getDigitByte(second / 10, 4) : getDigitByte(0, 4);
```
Comment (KenwoodFox) [inline firmware/lib/display/display.cpp:73]: Removed Edge Case Detection

File: firmware/lib/display/display.cpp [hunk 3]
```diff
@@ -70,10 +71,10 @@ void Display::setDispTime(uint8_t hour, uint8_t minute, uint8_t second, uint8_t
...
+        dispData = getDigitByte(second / 10, 4);
         break;
     case 5:
-        dispData = second < 60 ? getDigitByte(second % 10, 5) : getDigitByte(0, 5);
```
Comment (KenwoodFox) [inline firmware/lib/display/display.cpp:76]: Removed Edge Case Detection

File: firmware/lib/display/display.cpp [hunk 4]
```diff
@@ -56,9 +56,10 @@ void Display::updateBoard()
 void Display::setDispTime(uint8_t hour, uint8_t minute, uint8_t second, uint8_t tenths)
 {
     switch (currentSegment)
-    { // switch on digit location
+    { // Switch on digit location
     case 0:
-        dispData = getDigitByte(hour / 10, 0);
```
Comment (KenwoodFox) [inline firmware/lib/display/display.cpp:61]: Hour will dynamically not show leading 0

File: firmware/src/main.cpp [hunk 5]
```diff
@@ -174,13 +174,17 @@ void updateBoard(void)
   display.setCapture(!digitalRead(gpsSerialCheck)); // if the gps is being read from
   display.setHighSpec(isHighSpec());                // if the time has been locked in/synced to the rtc
 
-  display.setDispTime(isUsingLocalTZInput ? getUTCOffsetHours(hour()) : hour() % (clockFormat),
```
Comment (KenwoodFox) [inline firmware/src/main.cpp:177]: Consider breaking this up

**Comment by KenwoodFox** [comment]
Should merge soon and open a new PR to address #14

**Pull request merged**